lib/template-mixins.js

```diff
@@ -82,8 +82,23 @@ module.exports = function (fields, options) {
 
         var translate = options.translate || req.translate || _.identity;
 
-        var t = function (key) {
-            return translate(sharedTranslationsKey + key);
+        var t = function (key, prependSharedKey) {
+            if (prependSharedKey !== false) {
+                key = sharedTranslationsKey + key;
```

Comment:
Looking further up the code the `.` is appended to sharedTranslationsKey if absent